examples/iscsi-dd.c

```diff
@@ -758,7 +762,11 @@ int main(int argc, char *argv[])
 		}
 	}
 
-	clock_gettime(CLOCK_MONOTONIC, &end_time);
+	ret = clock_gettime(CLOCK_MONOTONIC, &end_time);
+	if (ret < 0) {
+		fprintf(stderr, "clock_gettime(CLOCK_MONOTONIC) failed\n");
+		exit(10);
+	}
```

Comment:
Does the action taken if clock_gettime() fails have to be this drastic? How about modifying show_perf() such that it does not show the transfer speed if clock_gettime() fails?

Reply:
Good idea - fixed.